### PR TITLE
Scale timeouts on Travis

### LINKED ISSUE #62: Fix flaky sampler test
`TestSamplerTicks` is flaky on Travis, likely because Travis is extremely slow. We should either extend the sleep to a few ticks, or we should add a tchannel-style timeout multiplier flag.

### PR DESCRIPTION
**Pull request by akshayjshah**
Fixes #62.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 language: go
+sudo: false
 go:
   - 1.5
   - 1.6
   - tip
 env:
   global:
-    - GO15VENDOREXPERIMENT=1
+    - GO15VENDOREXPERIMENT=1 TEST_TIMEOUT_SCALE=10
```
Comment (abhinav) [inline .travis.yml]: Can do:

```
- GO15VENDOREXPERIMENT=1
- TEST_TIMEOUT_SCALE=10
```

**Comment by abhinav** [comment]
lol :shipit:

**Comment by prashantv** [comment]
lgtm

File: .travis.yml (hunk 1 above)
Comment (akshayjshah) [inline .travis.yml]: Oh, cool - I didn't know that the global env vars are treated specially. Changed.

**Pull request merged**